example/example.ts

```diff
@@ -38,18 +43,103 @@ const changeExample = (selectedExample: string) => {
   }
 
   body.appendChild(jsonForms);
+
+};
+
+const changeStyle = (style) => {
+  $('select').material_select('destroy');
+  if (style === 'bootstrap') {
+    bootstrap();
+  } else if (style === 'materialize') {
+    material();
+  } else {
+    none();
+  }
 };
+
 window.onload = (ev) => {
+
+  // create select element for selecting the example to be displayed
   const examplesDiv = document.getElementById(exampleDivId);
```

Comment:
I agree, I've created a [follow-up issue](https://github.com/eclipsesource/jsonforms/issues/575) to track this.